src/ILVerify/src/TypeSystemHelpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+
+namespace ILVerify
+{
+    public static class TypeSystemHelpers
+    {
+        /// <summary>
+        /// Returns the "reduced type" based on the definition in the ECMA-335 standard (I.8.7).
+        /// </summary>
+        public static TypeDesc GetReducedType(this TypeDesc type)
```

Comment:
I meant change public to internal here. (The namespace - I do not really care about.)